### PR TITLE
New package: input-leap.input-leap version 3.0.1

### PR DESCRIPTION
**Pull request by KoljaFrahm**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/181393)

### CODE REVIEW

**Comment by KoljaFrahm** [comment]
Closes https://github.com/input-leap/input-leap/issues/2017

**Review by p12tic — APPROVED** [review]
LGTM, in case opinion of InputLeap maintainer is useful.

File: manifests/i/input-leap/input-leap/3.0.1/input-leap.input-leap.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: input-leap
+PublisherUrl: https://github.com/input-leap
+PublisherSupportUrl: https://github.com/input-leap/input-leap/issues
+PackageName: input-leap
+PackageUrl: https://github.com/input-leap/input-leap
+License: GNU General Public License v2.0
+LicenseUrl: https://github.com/input-leap/input-leap/blob/master/LICENSE
+ShortDescription: Input-leap is open-source KVM software
```
[the remark this answers is not in the document]
Comment (KoljaFrahm) [reply manifests/i/input-leap/input-leap/3.0.1/input-leap.input-leap.locale.en-US.yaml]: Can do

**Comment by stephengillie** [comment]
Hi @KoljaFrahm,

Verify:
image
image
Should `Apple.Bonjour` be a dependency?
image
The `PackageVersion` should have the `-release` added, so it matches the Registry entry.

**Comment by KoljaFrahm** [comment]
@stephengillie I changed the version and added bonjour as requirement

**Comment by KoljaFrahm** [comment]
Removed Version 3.0.2 again

**Review by stephengillie — APPROVED** [review]

**Pull request merged**